lms/views/predicates/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""
+Custom Pyramid view predicates.
+
+See:
+
+https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/hooks.html#view-and-route-predicates
+"""
+from lms.views.predicates._lti_launch import DBConfigured
+from lms.views.predicates._lti_launch import CanvasFile
+from lms.views.predicates._lti_launch import URLConfigured
+from lms.views.predicates._lti_launch import Configured
+from lms.views.predicates._lti_launch import AuthorizedToConfigureAssignments
```

Comment:
[PEP 8 allows either form](https://www.python.org/dev/peps/pep-0008/#imports) but I think you're right that most people would prefer the single-line version. I think it can make grepping the code for all uses of foo more difficult though